starlette/routing.py

```diff
@@ -788,7 +798,10 @@ def add_route(
         self.routes.append(route)
 
     def add_websocket_route(
-        self, path: str, endpoint: typing.Callable, name: typing.Optional[str] = None
+        self,
+        path: str,
+        endpoint: typing.Callable[..., typing.Any],
```

Comment:
Isn't an endpoint always `Callable[[WebSocket], typing.Any]`?

Reply:
See https://github.com/encode/starlette/pull/2180#discussion_r1248772844 (same for https://github.com/encode/starlette/pull/2180#discussion_r1247700951, https://github.com/encode/starlette/pull/2180#discussion_r1247701050)